app/controllers/api/v1/doctors_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Api::V1::DoctorsController < ApplicationController
+  def index
+    doctors = Doctor.all
+    render json: doctors_representer(doctors).as_json(only: %i[id name specialization bio image experience
+                                                               rate])
+  end
+
+  def show
+    @doctor = Doctor.find_by(id: params[:id])
+    if @doctor
+      render json: @doctor, status: 200
+    else
+      render json: {
+        error: 'Doctor not found', status: 404
+      }
+    end
+  end
+
+  def create
+    @doctor = Doctor.new(doctor_params)
+
+    if @doctor.save
+      render json: @doctor, status: 200
+    else
+      render json: {
+        error: 'Error creating service...'
+      }
+    end
+  end
+
+  def destroy
+    @doctor = Doctor.find(params[:id])
+    @doctor.destroy
+    head :no_content
```

Comment:
What happens if there is an error when performs the first two tasks? Don't foget your error handling